### ISSUE TITLE
Link colour can be hard to read in certain lights

### ISSUE THREAD

**Issue by rsmith013** [cedadev/datamad2#215]

**Comment by rsmith013** [comment]
Look at accessibility guides for guidance on link colour

**Comment by rsmith013** [comment]
Accessibility guidance says that foreground/background colours should have a contrast ratio of at least 3:1 to be properly legible. The current link text colour has a contrast ratio of 2.04:1

This is a good tool that will calculate the contrast measurehttps://juicystudio.com/services/luminositycontrastratio.php#specify
